spec/integration/recipes/local.rb

```diff
@@ -0,0 +1,10 @@
+package 'sl' do
+  version '3.03-17+b2'
```

Comment:
`ruby` docker image does not have `sl` package version `3.03-17`, it only has `3.03-17+b2`.